lib/rubocop/thread_safety/plugin.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'lint_roller'
+
+module RuboCop
+  module ThreadSafety
+    # A plugin that integrates RuboCop ThreadSafety with RuboCop's plugin system.
+    class Plugin < LintRoller::Plugin
+      # :nocov:
+      def about
+        LintRoller::About.new(
+          name: 'rubocop-thread_safety',
+          version: Version::STRING,
+          homepage: 'https://github.com/rubocop/rubocop-thread_safety',
+          description: 'Code style checking for ThreadSafety test files.'
```

Comment:
This wording seems a bit odd to me. What does "ThreadSafety test files" mean?

Reply:
Sorry, that’s a copy/paste error on my part. I’ve changed it to

```suggestion
          description: 'Thread-safety checks via static analysis.'
```